.gitignore

```diff
@@ -91,3 +91,7 @@ tags
 .tags
 !tags/
 gtags.files
+
+eln-cache/
+meghanada/
+vimish-fold/
```

Comment:
Make sure you add a newline at the end of file, otherwise :+1:

Reply:
@sdwolfz fixed